scripts/check-links.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Should this be `#!/bin/sh`?

Reply:
@jorgeorpinel Can you try and let me know?